src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,13 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs
+  * @constructor Create a new command reader
+  */
+case class CommandReader(inputs: Array[String]) {
+  assert(inputs.length >= 2)
```

Comment:
I think a better approach here would be to print out a helpful message that explains what the arguments should be if the number of arguments is too small. It's more helpful to the end user, since they can then just run our program without arguments to get a usage message.